debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -573,8 +573,9 @@ protected void assertSourceInfo(SourceRecord record) {
         assertNotNull(source.getString("table"));
     }
 
-    protected static String tableNameFromInsertStmt(String statement) {
-        return tableIdFromInsertStmt(statement).toString();
+    protected static String topicNameFromInsertStmt(String statement) {
+        String qualifiedTableNameName = tableIdFromInsertStmt(statement).toString();
+        return qualifiedTableNameName.replaceAll("[ \"]", "_");
```

Comment:
@gunnarmorling I am not sure I understand this change

Reply:
The issue was that expected messages for the tests must be organized by topic and not by table (as we'll receive topic names in the actual records and we'd not have a way to conclude the original table name easily, as it's a one-way function from table to topic name). Hence this method derives the topic name under which the changes for a given INSERT are to be expected.